ci/install.sh

```diff
@@ -35,10 +35,8 @@ if [ ! -z "${CRICTL_VERSION}" ]; then
     mv bin/crictl bin/crictl-${CRICTL_VERSION}
   fi
   cp bin/crictl-${CRICTL_VERSION} bin/crictl
-  # kubeadm being run by minikube complains that it fails to find 'crictl' in
-  # the system path on Travic CI jobs when using K8s 1.11, even if I add it like
-  # below ahead of time...
-  sudo cp bin/crictl-${CRICTL_VERSION} /usr/bin
+  # minikube is run with sudo so the modified PATH is lost
+  sudo ln -s "${PWD}/bin/crictl-${CRICTL_VERSION}" /usr/bin/crictl
```

Comment:
```suggestion
  sudo ln -s "bin/crictl-${CRICTL_VERSION}" /usr/bin/crictl
```

For my own comprehension, this would work as well right? Because invoking sudo won't change the current working directory right?

Reply:
Not sure! But if anything else used tried to use `/usr/bin/crictl` it would be broken. An alternative would be to set `PATH` as an argument to `sudo`.